README.md

```diff
@@ -23,26 +26,49 @@ The Kubernetes segment uses a couple of custom highlight groups. You'll need to
 ```json
 {
   "groups": {
-    "kubernetes":                { "fg": "gray8",           "bg": "darkestblue", "attrs": [] },
-    "kubernetes:divider":        { "fg": "gray4",           "bg": "darkestblue", "attrs": [] }
+    "kubernetes_cluster":         { "fg": "gray10", "bg": "darkestblue", "attrs": [] },
+    "kubernetes_cluster:alert":   { "fg": "gray10", "bg": "darkestred",  "attrs": [] },
+    "kubernetes_namespace":       { "fg": "gray10", "bg": "darkestblue", "attrs": [] },
+    "kubernetes_namespace:alert": { "fg": "gray10", "bg": "darkred",     "attrs": [] },
+    "kubernetes:divider":         { "fg": "gray4",  "bg": "darkestblue", "attrs": [] },
   }
 }
 ```
 
 Then you can activate the Kubernetes segment by adding it to your segment configuration, for example in `.config/powerline/themes/shell/default.json`:
 
-```json
+```javascript
 {
     "function": "powerline_kubernetes.kubernetes",
-    "priority": 30
+    "priority": 30,
+    "args": {
+        "show_kube_logo": true, // set to false to omit the Kube logo
+        "show_cluster": true, // show cluster name
+        "show_namespace": true, // show namespace name
+        "show_default_namespace": false, // do not show namespace name if it's "default"
+        "alerts": [
+          "live", // show line in different color when namespace matches
+          "cluster:live"  // show line in different color when cluster name and namespace match
+        ]
+    }
 }
 ```
 
 By default the segment will look for the Kubernetes config under `~/.kube/config`.
+
+
 ## License
 
 Licensed under the [MIT License](LICENSE).
 
+
+## Authors
+
+Created by [so0k](https://github.com/so0k/). Code contributions by:
+- [bokysan](https://github.com/bokysan)
+
+
 ---
 
 Inspired by [powerline-docker](https://github.com/adrianmo/powerline-docker).
+Inspired by [powerkube](https://github.com/zcmarine/powerkube).
```

Comment:
this is not correct :D

https://github.com/so0k/powerline-kubernetes/commit/f1805a0d054f6bf076955de4c0c22f7881aeae38
https://github.com/zcmarine/powerkube/commit/a95986a24f2057727f3019f0245c15af5e64be4f

Reply:
I have removed this line from my master branch, although I did get ideas there (e.g. red color for important contexts).

Reply:
(I'm not quite sure why github is not showing my latest commit on this pull request, though)

Reply:
ooh, I see what you mean